### ISSUE TITLE
missing X11

### ISSUE THREAD

**Issue by Andrewmoser9** [dilevin/computer-graphics-bounding-volume-hierarchy#115]
<img width="658" alt="Capture" src="https://github.com/user-attachments/assets/e17cc663-e40c-40ae-ab7f-776aac398669">

**Comment by Zhecheng-Wang** [comment]
#114

**Comment by Andrewmoser9** [comment]
Can we meet in person? I am struggling with opening Xming for some reason

**Comment by Zhecheng-Wang** [comment]
No, I have a paper deadline coming up. Did you try to build A4 on Windows? I understand you might want to test on teach.cs machines to see if it works on Linux. But since I am grading A4, the chance of code written for a Windows machine can't be compiled for Linux is next to zero.

**Comment by Andrewmoser9** [comment]
Okay no worries sorry I am really bad with setup things like this My output looks correct I will try asking around thank you so much.

**Comment by jennicao** [comment]
Any X server works if you’re having trouble with Xming, I used Vcxsrv.
